src/Entity/ProductCommentGrade.php

```diff
@@ -37,7 +37,7 @@ class ProductCommentGrade
     /**
      * @ORM\Id
      * @ORM\ManyToOne(targetEntity="ProductComment")
-     * @ORM\JoinColumn(name="id_product_comment", referencedColumnName="id_product_comment")
+     * @ORM\JoinColumn(name="id_product_comment", referencedColumnName="id_product_comment", onDelete="CASCADE")
```

Comment:
Because of this Doctrine magic, we can remove the manual Grade deletion at https://github.com/PrestaShop/productcomments/blob/dev/src/Repository/ProductCommentRepository.php#L117
and mark the corresponding function as Deprecated too https://github.com/PrestaShop/productcomments/blob/dev/src/Repository/ProductCommentRepository.php#L560